### PR TITLE
Allow i18n query locale to be configurable

### PR DESCRIPTION
**Pull request by adelowo**
#71

### CODE REVIEW

File: param.go [hunk 1]
```diff
@@ -68,8 +68,11 @@ func (e *engine) parseRequestParams(ctx *Context) {
 		}(req)
 	}
 
-	// i18n option override by Query parameter `lang`
-	if lang := ctx.Req.QueryValue(keyOverrideLocale); !ess.IsStrEmpty(lang) {
+	// i18n option via the config value "i18n.url_param_name".
+	//If that value is missing, we default to the `lang` query parameter
+	queryParam := AppConfig().StringDefault("i18n.url_param_name", keyOverrideLocale)
+
```
Comment (jeevatkm) [inline param.go]: Please do not added unnecessary newline.

File: param.go [hunk 2]
```diff
@@ -68,8 +68,11 @@ func (e *engine) parseRequestParams(ctx *Context) {
 		}(req)
 	}
 
-	// i18n option override by Query parameter `lang`
-	if lang := ctx.Req.QueryValue(keyOverrideLocale); !ess.IsStrEmpty(lang) {
+	// i18n option via the config value "i18n.url_param_name".
+	//If that value is missing, we default to the `lang` query parameter
```
Comment (jeevatkm) [inline param.go]: Kindly use `goimports` or `goreturns` to format the source code.

File: param.go (hunk 1 above)
Comment (adelowo) [inline param.go]: Sorry about that.. I am used to that (for readability), would take a while to get used too. Would fix

Comment (adelowo) [inline param.go]: Done

**Pull request merged**